src/assets/scripts/client/constants/canvasConstants.js

```diff
@@ -14,3 +14,21 @@ export const DEFAULT_CANVAS_SIZE = {
     HEIGHT: 480,
     WIDTH: 640
 };
+
+/**
+ * Various options for drawing aircraft on canvas
+ *
+ * @enum AIRCRAFT_DRAW_OPTIONS
+ * @type {Object}
+ * @final
+ */
+export const AIRCRAFT_DRAW_OPTIONS = {
+    /**
+     * Length of PTL lines, in minutes. Standard is 1.
```

Comment:
Sorry, second nature. "Projected Track Lines" is the proper name; it's a setting within the DCB menu of a STARS scope (used in modern FAA TRACONs). I'll amend it to that.